recipes-core/initrdscripts/init-nilrt-ramfs.bb

```diff
@@ -2,6 +2,10 @@ SUMMARY = "NILRT initramfs init script"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
 
+
+PV = "1.0"
```

Comment:
These four files:

    recipes-core/initrdscripts/init-nilrt-ramfs.bb
    recipes-core/initrdscripts/init-restore-mode.bb
    recipes-xfce/xfce-nilrt-settings/xfce-nilrt-settings.bb
    recipes-xfce/xserver-xfce-init/xserver-xfce-init.bb

each declare their versions to be 1.0, though the commit mentions that that is the default and simply renames most recipes.
Are these four files special in some way?

Reply:
TBH because I started with those files and added the PV variable. Then realized that it didn't matter and would take 3 times as long to do it to all the files, so I just stopped. I can go back and remove them if you have some objection to them being defined, but it doesn't make a technical difference.